examples/http-proxy/README.md

```diff
@@ -46,7 +43,7 @@ libp2p-peer addresses:
 The run the local peer, indicating that it will need to forward http requests to the remote peer as follows:
 
 ```
-$ ./http-proxy -d /ip4/127.0.0.1/tcp/12000/ipfs/QmddTrQXhA9AkCpXPTkcY7e22NK73TwkUms3a44DhTKJTD
+> ./http-proxy -d /ip4/127.0.0.1/tcp/12000/ipfs/QmddTrQXhA9AkCpXPTkcY7e22NK73TwkUms3a44DhTKJTD
```

Comment:
(:roll_eyes:) Unfortunately, that argument kind of makes sense. `$` for variables, `#` for comments, `>` for prompts.


---

But really, 

```bash
𝝺: echo "there can be only one"
```